mindsdb/interfaces/knowledge_base/controller.py

```diff
@@ -97,6 +97,17 @@ def get_reranking_model_from_params(reranking_model_params: dict):
     return BaseLLMReranker(**params_copy)
 
 
+def safe_pandas_is_datetime(value: str) -> bool:
+    """
+    Check if the value can be parsed as a datetime.
+    """
+    try:
+        result = pd.api.types.is_datetime64_any_dtype(value)
```

Comment:
`is_datetime64_any_dtype` accepts dtype, not value
https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_datetime64_any_dtype.html